Dockerfile

```diff
@@ -170,6 +171,23 @@ RUN if [ ! -z "${TIMESCALE_PROMETHEUS}" ]; then \
         done; \
     fi
 
+ARG TIMESCALE_PROMSCALE_EXTENSION=
+# build and install the promscale_extension extension
+RUN if [ ! -z "${TIMESCALE_PROMSCALE_EXTENSION}" ]; then \
+        curl https://sh.rustup.rs -sSf | bash -s -- -y \
```

Comment:
wonder if we could save time by not having to redo this now while the other extension lives

Reply:
Takes only a few seconds, it does not redownload all required tools.